cmd/test_util.go

```diff
@@ -159,6 +159,21 @@ func compareTextFiles(file1, file2 string) bool {
 		return false
 	}
 
+	f1stat, err := f1.Stat()
```

Comment:
The problem is this doesn't give you any idea where they diff. The other code gave you a diff, this doesn't anymore.

Reply:
I've moved the stat code to after the scan loop, now we'll get where the files don't match as well as it'd match length of both files at the end. Do you have a better way around this? One thing I thought of was to just set s1 to be the scanner of the larger file, but anyways if files dont match and we scan doesnt output an error but stat does we can imply that files don't match in length hence files missing some content after?

Reply:
It's not the cleanest solution, but I think given how complicated a different solution would be this is I guess the best solution.